src/Pool.sol

```diff
@@ -688,81 +451,7 @@ abstract contract Pool {
         amtDeltas[thisCycle].nextCycle += int128(uint128(nextCycleSecs)) * amtPerSecDelta;
     }
 
-    /// @notice Stops proxy of `msg.sender` for the duration of the modified function.
-    /// This removes and then restores any effects of the proxy on all of its receivers' futures.
-    /// It allows the function to safely modify any properties of the proxy
-    /// without having to updating the state of its receivers.
-    modifier suspendProxy {
```

Comment:
Ah nice, that this is removed. In Solidity it is anyway kind of standard to use modifiers only for access-control